tests/phpunit/tests/block-supports/layout.php

```diff
@@ -181,6 +181,7 @@ public function test_outer_container_not_restored_for_aligned_image_block_with_t
 	 * @param string $expected_output The expected output.
 	 */
 	public function test_layout_support_flag_renders_classnames_on_wrapper( $args, $expected_output ) {
```

Comment:
I notice this function includes references to the tickets directly above it. Should the linked trac ticket be added to that list?